SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2156,9 +2166,46 @@ bool USpatialNetDriver::FindAndDispatchStartupOpsServer(const TArray<Worker_OpLi
 		}
 	}
 
+	auto IsVirtualWorkerTranslatorReady = [this]()
```

Comment:
Why make this a lambda instead of a bool. I don't think delaying execution gets us anything.

Reply:
What's the drawback of the lambda?

Reply:
Looking at it again there is no reason for it to be a lambda :P

Reply:
That one. It's not wrong it's just unnecessary code.

Reply:
Removed